src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -251,8 +251,14 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
       this.metricsPromise = undefined;
     }
 
-    // Just return if the metric types are unset, there is no data, or charts are unsupported
-    if (!destMetricType || !sourceMetricType || !this.hasSupportedCharts(edge) || (!isGrpc && !isHttp && !isTcp)) {
+    // Just return if the metric types are unset, there is no data, destination node is "unknown" or charts are unsupported
+    if (
+      !destMetricType ||
+      !sourceMetricType ||
+      !this.hasSupportedCharts(edge) ||
+      (!isGrpc && !isHttp && !isTcp) ||
+      destData.isInaccessible
```

Comment:
I don't think this is right.  You can have traffic to an inaccessible node.  Inaccessible just means that the user can't navigate to or see it.  You may be able to just get rid of this condition. If that causes a problem rendering the edge chart then you may need to update getNodeMetricType() such that for nodeType SERVICE the metricType is WORKLOAD when the service is unknown.

Reply:
See the other reply... I think the same applies to this and is probably correct to have the `isInaccessible` condition to avoid sending a request that will return with error.